Directory.Build.props

```diff
@@ -2,11 +2,12 @@
   <PropertyGroup>
     <!-- This property will be set by the test runner.
          You can change it locally when running without the test runner to match with the .NET SDK under test. -->
-    <TestTargetFramework Condition="'$(TestTargetFramework)' == ''">net6.0</TestTargetFramework>
+    <TestTargetFramework Condition="'$(TestTargetFramework)' == ''">net8.0</TestTargetFramework>
 
     <SystemDataOdbcVersion>8.0.0</SystemDataOdbcVersion>
-    <MicrosoftNetTestSdkVersion>17.10.0</MicrosoftNetTestSdkVersion>
-    <XunitVersion>2.8.1</XunitVersion>
-    <XunitRunnerVisualStudioVersion>2.8.1</XunitRunnerVisualStudioVersion>
+    <MicrosoftNetTestSdkVersion>17.13.0</MicrosoftNetTestSdkVersion>
+    <XunitVersion>2.9.3</XunitVersion>
+    <XunitV3Version>2.0.0</XunitV3Version>
+    <XunitRunnerVisualStudioVersion>3.0.2</XunitRunnerVisualStudioVersion>
```

Comment:
Besides adding the XunitV3Version, this updates versions to the latest stable nuget.org version for these packages.